example_template.md

```diff
@@ -0,0 +1,102 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+- [Run the EXAMPLE NAME](#run-the-example-name)
+  - [🗝️ Expected knowledge](#Expected-knowledges)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌀 Flow Diagram](#flow-diagram)
+  - [🌟 Results](#results)
+  - [🧞 OPTIONAL: EXTRA KNOWLEDGE FROM THIS EXAMPLE](#optional)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Run the EXAMPLE NAME
+
+Write a short description of
+1. What is this?
+2. What is the expected outcome?
+3. What the user will learn?
+
+If you want to run the example with Docker, check the instruction at the end of this README.  
+
+## 🗝️ Expected knowledge
+
+Write a list of whatever the user is expected to know.
+
+1. Read up on [Jina 101](http://101.jina.ai).
+2. Python knowledge.
+
+### Install requirements
+
+There are some requirements to run this example, to download everything you need, just run in your terminal:
+
+```
+pip install -r requirements.txt
+```
+
+## 🔮 Overview of the files
+
+Add a list with all files in the example:
+
+* index.yml: YAML file for indexing
+* query.yml: YAML file for querying
+* encoder.yml: YAML file for encoder pod
+* /workspace: Directory that stores indexed files (embeddings and documents). Automatically created after the first indexing.
+
+## 🏃 Run the Flows
```

Comment:
I would "step" to each of these titles. 

Step 1. Run the Index Flow 
Step 2. Run the Query Flow 

https://developers.google.com/tech-writing/one/lists-and-tables